### PR TITLE
Add Hackers Toolbox pages

### PR DESCRIPTION
**Pull request by ravern**
## Changes

* Renames existing references to Hacker Tools to Hackers Toolbox (**except for past event posts**, to avoid changing permalinks)
* Adds a dedicated page at `/hackerstoolbox`
* Adds a schedule page at `/hackerstoolbox/schedule`

### CODE REVIEW

File: layouts/_default/hackerstoolbox_schedule.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{{ define "main" }}
+{{ $hs := index $.Site.Data.hackers_toolbox
+(partial "projects/get_project.html" (dict "projectName" "hackers_toolbox")) }}
+<main class="container main">
+  <div class="events hs col-12 col-md-10 offset-md-1">
+    <h1>Hackers Toolbox Schedule (AY{{ $.Site.Params.academicYear }} Semester {{ $.Site.Params.semester }})</h1>
```
Comment (francisyzy) [inline layouts/_default/hackerstoolbox_schedule.html:6]: I guess this section will automatically update to `AY2324 Semester 1` once the semester starts?

image

Comment (shenyih0ng) [reply layouts/_default/hackerstoolbox_schedule.html:6]: we are keeping track of the current academic year and semester in `config.toml`, once that is updated data will be pulled from data yaml files corresponding to the current semester

**Review by francisyzy — APPROVED** [review]

File: content/hackerstoolbox/_index.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+[Check out our event schedule here]({{< ref "/hackerstoolbox/schedule" >}}) and sign up for our next event today! 
+
+## FAQ
+
+### I didn't receive the zoom link despite signing up via the RSVP form
+
+If you do not receive any follow up emails with the event's zoom link, check your spam/junk mail folder. If it is there, marking the email as "Not Spam" will help to prevent our emails from being directed to spam/junk in future.
+
+If you really have not received any email, [contact us]({{< ref "/contact" >}}) and we will try to assist you as best we can.
```
Comment (shenyih0ng) [inline content/hackerstoolbox/_index.md:21]: can probably remove this, don't think this is relevant anymore

File: content/hackerstoolbox/_index.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+title: About Hackers Toolbox
+description: About Hackers Toolbox
+date: "2023-06-23"
+---
+
```
Comment (shenyih0ng) [inline content/hackerstoolbox/_index.md:6]: add some pics maybe?

Comment (ravern) [reply content/hackerstoolbox/_index.md:6]: No media from past sessions really suitable to use for a banner image, will upload one once we have our first session this semester and take some pics.

**Pull request merged**